scrapers/MeanBitches.yml

```diff
@@ -0,0 +1,56 @@
+name: MeanBitches
+
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - https://www.meanbitches.com/tour/models/
+    scraper: performerScraper
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://www.meanbitches.com/tour/updates/
+    scraper: sceneScraper
+
+
+xPathScrapers:
+  performerScraper:
+    performer:
+      Name: //div[contains(@class,'title_bar')][1]/text()            
+      Image:
+        selector: //tr/td/img[@class="thumbs"]/@src
+        postProcess:
+          - replace:
+            - regex: (.*)
+              with: http://meanbitches.com/$1
+  sceneScraper:
+    common:
+      $titlebar: //table[@id="episode_title"]/tbody/tr/
+    scene:
+      Title: //table[@id="episode_title"]/tbody/tr/td[1]/text()
+      Details: //div[@class="episode_description"]/text()
+      Studio:
+        Name: 
+          fixed: Mean Bitches
+        URL:
+          fixed: https://www.meanbitches.com/
+      Performers:
+        Name: 
+          selector: $titlebar/td[2]/text()
+          postProcess:
+            - replace:
+                - regex: "Featuring:(.*)"
+                  with: $1
+      Image: 
+        selector: //a[@class='trailer_link']/img/@src
+        postProcess:
+          - replace:
+            - regex: (.*)
+              with: http://meanbitches.com/$1
```

Comment:
Some scenes dont have a trailer link ( the sample scene from above for example)
```yaml
      Image:
        selector: //a[@class='trailer_link']/img/@src|//td[@class="episode_thumbs"][@align="center"]//img/@src
        postProcess:
          - replace:
              - regex: ^/
                with: http://meanbitches.com/
```
Looks ok